portal-api/src/v1/users/validation/rules/firstName-must-not-be-empty.test.js

```diff
@@ -0,0 +1,29 @@
+const firstNameMustNotBeEmpty = require('./firstName-must-not-be-empty');
+
+describe('firstNameMustNotBeEmpty', () => {
+  const firstNameMustNotBeEmptyError = [
+    {
+      firstname: {
+        order: '1',
+        text: 'First name is required',
+      },
+    },
+  ];
+
+  const testCases = [{ description: 'when no first name is provided', change: { firstName: '' } }];
```

Comment:
Please add more assertions to `testCases` since `.each` is being used, please add:
* null
* undefined
* []
* {}
* ' '

Reply:
Added more assertion with above combination.